.env.example

```diff
@@ -1,4 +1,6 @@
 NEYNAR_API_KEY=
+# an approved signer uuid to send cast
+NENYAR_SIGNER=
```

Comment:
called NEYNAR_SIGNER_ID in the app/libs/farcaster/respondReimbursementCast.ts file. I assume these refer to the same env variable? Please align them :)

Reply:
ohh, my bad. would fix now